### PR TITLE
[zh-hans] Align version for typespec, collections, documentation, mod…

### PR DESCRIPTION
**Pull request by kenspirit**
…ules, mnesia, plug

### CODE REVIEW

**Comment by doomspork** [comment]
Thank you @kenspirit! I hope all is well with you and yours 😁

**Comment by kenspirit** [comment]
> Thank you @kenspirit! I hope all is well with you and yours 😁

Welcome @doomspork .  It's Chinese New Year here and we are fantastic.  So the review is probably not responded that promptly.  😁

**Comment by doomspork** [comment]
Happy New Year 🎉 

No rush on reviews. I hope you all have a wonderful celebration!

File: zh-hans/lessons/basics/modules.md [hunk 1]
```diff
@@ -103,6 +103,35 @@ iex> %{name: "Sean"} = sean
 #Example.User<name: "Sean", roles: [...], ...>
 ```
 
+到了 Elixir 1.8，结构体允许包含自定义的审视方式。以下通过查看 `sean` 结构来理解这是如何实现的：
```
Comment (zillou) [inline zh-hans/lessons/basics/modules.md]: `检查`是不是比`审视`更常用一点儿？

File: zh-hans/lessons/libraries/distillery.md [hunk 2]
```diff
@@ -0,0 +1,238 @@
+---
+version: 2.0.0
+title: Distillery (基础)
+---
+
+Distillery
```
Comment (zillou) [inline zh-hans/lessons/libraries/distillery.md]: 我猜英文中这一行是多余的。

File: zh-hans/lessons/libraries/distillery.md [hunk 3]
```diff
@@ -0,0 +1,238 @@
...
+* /lib 文件夹
+  * 这包含了应用编译后的字节码，及其所有依赖。
+* /releases 文件夹
+  * 这包含了发布包，及其钩子和自定义命令的元数据。
+* /erts-VERSION
+  * 这是 Erlang 运行时环境。它可以让你的机器在没有安装 Erlang 或 Elixir 的情况下运行你的应用。
+
+### 入门及安装
+
+把 Distillery 当作依赖，添加到你项目里 的 `mix.exs` 文件里头。*注意* —— 如果你的是 unbrella 应用，请把它添加到项目根目录的 mix.exs 文件里。
```
Comment (zillou) [inline zh-hans/lessons/libraries/distillery.md]: ~unbrella~ umbrella

File: zh-hans/lessons/libraries/distillery.md [hunk 4]
```diff
@@ -0,0 +1,238 @@
...
+    > _build/dev/rel/book_app/bin/book_app attach
+
+For a complete listing of commands and their use:
+
+    > _build/dev/rel/book_app/bin/book_app help
+```
+
+在命令行输入命令 ` _build/dev/rel/MYAPP/bin/MYAPP foreground` 就可以启动你的应用。当然，你需要把 MYAPP 替换为你的项目名称。这样我们就已经通过发布包来运行我们的应用了！
+
+## 结合 Phoenix 来使用 distillery
```
Comment (zillou) [inline zh-hans/lessons/libraries/distillery.md]: 我更觉得这个是在 Phoenix 项目中使用 distillery。:confused:

**Comment by zillou** [comment]
春节快乐，@kenspirit ! 

感谢你春节期间还在更新课程内容。 我提了一点小问题，请你参考。

File: zh-hans/lessons/libraries/distillery.md (hunk 2 above)
Comment (kenspirit) [reply zh-hans/lessons/libraries/distillery.md]: > 我猜英文中这一行是多余的。

我也感觉是。

**Comment by kenspirit** [comment]
> 春节快乐，@kenspirit !
> 
> 感谢你春节期间还在更新课程内容。 我提了一点小问题，请你参考。

都改好了。

**Pull request merged**